### DISCUSSION TITLE
Printing results

### DISCUSSION THREAD

**Discussion by ghost** [enlightn/enlightn#28] [Ideas]
Hi,

I'm trying your tool now and it seems awesome for development - thank for that :).

However I have an question. Is / Will be there any way how to print results to some file log (for better sharing in team or for better checking)?

Something like `php artisan enlightn -p`

Thanks for answer and good luck.

**Comment by paras-malhotra** [comment]
Hi @tomve, we don't have a feature to share results for now but it's definitely in the pipeline! Keep watching for new releases, we're planning on developing this feature over the next 2 months or so.

**Reply by ghost** [reply]
I will for sure.

Good luck

**Reply by paras-malhotra** [reply]
@tomve, we released this feature a few weeks ago. Just wanted to update you. :)
